dvc/rwlock.py

```diff
@@ -73,8 +73,11 @@ def _check_no_writers(lock, info, path_infos):
             continue
 
         raise LockError(
-            "'{}' is busy, it is being written to by:\n{}".format(
-                str(path_info), _infos_to_str(writers)
+            "'{path}' is busy, it is being written to by:\n{process}"
+            "\n"
+            "If that is not the case, manually remove '.dvc/tmp/rwlock'"
```

Comment:
do we still have a global lock file at all? or it is a single file? 

I would make a section in the troubleshooting guide that describes this case and put a link from this error, along with the case when `rwlock` is corrupted for some reason. 

"... if that is not the case" is not very descriptive.

Reply:
It is a global lock file where we keep track of the processes in a JSON.

Reply:
I mean, is it the only file we use now? we used to have a global `lock` file as well. (and this one technically even being a single file is not a global lock - it introduced granular locks the way I understand this).

Reply:
@shcheklein , yes, single file, granular lock.

Reply:
@shcheklein , so, introducing an entry to the troubleshooting guide would make it for you?

What about: ""If that is not the case" -> If the process doesn't exist anymore"

Reply:
so, we don't use `lock` file anymore, right? only `rwlock` now and this error happens only if rwlock for a specific path is busy?

`If the process doesn't exist anymore` looks better!

I would say if it's the only reason for LockError now, no `lock` files anymore, we don't need any any docs indeed.

Reply:
We do use `.dvc/lock` as a fallback :slightly_smiling_face: 

That's right, this error happens when a specific path is busy.

Reply:
what is the `fallback` logic here? 

is it the same exception thrown when `.dvc/lock` exists?

Reply:
My bad, @shcheklein , it is not a fallback but a configurable option.
So, there's a `make_lock` method that we use to create locks and you can choose between rwlock and hardlink-lock.

But is either one or the other.

We have the following message for the hardlink based lock:
```python
    "cannot perform the command because another DVC process seems to be "
    "running on this project. If that is not the case, manually remove "
    "`.dvc/lock` and try again."
```

Reply:
got it, thanks @mroutis !